app/controllers/concerns/effective_user.rb

```diff
@@ -3,11 +3,11 @@ def effective_user_id
     [
       session[:ial2_recovery_user_id],
       session[:doc_capture_user_id],
-      current_user&.id,
     ].find(&:present?)
   end
 
   def effective_user
+    return current_user if current_user
```

Comment:
technically this does not behave the same way because the recover and capture session ids will override current_user.  i'm not sure if this is a problem

Reply:
ie the user has a different user signed in on phone but uses the link sent to mobile phone for a different user

Reply:
ah, I see, I will reorder